### ISSUE TITLE
Zeus crashes while executing GetNodeInfo API request

### ISSUE THREAD

**Issue by yo-mr-white** [ZeusLN/zeus#1423]
### Describe the bug

I am using latest version of Zeus (0.7.4) on iOS 16.3.1. I am connected to my node (LND v0.16.0-beta) via REST API and clearnet within local network.

If I want to view a list of channels or get details about a particular forward, app crashes. In LND logs, there is an error, that node could not be found (6x).

`[ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node`

It probably started to appear after the LND upgrade to LND v0.16.0-beta, but I am not 100% sure.

I tried uninstalling the app, adding the node configuration again, restarting LND - nothing helps. Unfortunately, there is no information in the LND logs about which specific node Zeus tried to search for and did not find.

### Reproduce

- Install LND v0.16.0-beta
- Install Zeus 0.7.4
- Connect to LND via REST API and clearnet
- List channels

### Zeus version

v0.7.4

### Node interface

LND (REST)

### Network

Clearnet

### Device

iPhone

### Device operating system

iOS 16.3.1

### Log output

```shell
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.278 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.282 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.305 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.315 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.315 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
Apr 04 14:10:43 heisenberg-ln lnd[95431]: 2023-04-04 14:10:43.316 [ERR] RPCS: [/lnrpc.Lightning/GetNodeInfo]: rpc error: code = NotFound desc = unable to find node
```

**Comment by kaloudis** [comment]
We think there is a data corruption issue on iOS. We have a new TestFlight build that we're trying out right now that should help address this. Reach out with your Apple ID if you'd like to help test.

**Comment by yo-mr-white** [comment]
Resolved by upgrading to 0.7.5 via Testflight. Thank you
